deploy_polygon/02_asset/02_deploy_asset_signed_auction_auth_v2.ts

```diff
@@ -0,0 +1,37 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
not sure where to put this file: deployment_polygon -> together with the v1 or in deployment?

Reply:
we can actually replace the currenct AssetSignedAuctionAuth since it's not used yet (v2 not needed)
and you deploy it only on L1

And maybe to stick with the naming of EstateSaleWithAuth, we can call it AssetSignedAuctionWithAuth